staging/src/k8s.io/kubectl/pkg/cmd/get/get_flags.go

```diff
@@ -73,6 +74,7 @@ func (f *PrintFlags) AllowedFormats() []string {
 	formats = append(formats, f.TemplateFlags.AllowedFormats()...)
 	formats = append(formats, f.CustomColumnsFlags.AllowedFormats()...)
 	formats = append(formats, f.HumanReadableFlags.AllowedFormats()...)
+	sort.Strings(formats)
```

Comment:
As I'm looking at it, I've noticed we're usually sorting inside each of the above `AllowedFormats` calls, see [GoTemplatePrintFlags](https://github.com/kubernetes/kubernetes/blob/52cd4d53ac8ac33cacd3d2c618cf22f1e3fd6a7e/staging/src/k8s.io/cli-runtime/pkg/genericclioptions/template_flags.go#L56) or [JSONPathPrintFlags](https://github.com/kubernetes/kubernetes/blob/52cd4d53ac8ac33cacd3d2c618cf22f1e3fd6a7e/staging/src/k8s.io/cli-runtime/pkg/genericclioptions/jsonpath_flags.go#L55) can you move this sort invocation over to [CustomColumnPrintFlags](https://github.com/kubernetes/kubernetes/blob/52cd4d53ac8ac33cacd3d2c618cf22f1e3fd6a7e/staging/src/k8s.io/kubectl/pkg/cmd/get/customcolumn_flags.go#L43) this will ensure that consumers of that structure will also get a consistent sorting of the flags and it'll bubble up automatically.

Reply:
Thanks. I finished.